docs/reference/content/guides/annotations.md

```diff
@@ -0,0 +1,292 @@
+| value() | List of fields (prepended with "-" for desc; defaults to asc). |  
+| name() | The name of the index |
+| unique() | Requires values in the index to be unique |
+| dropDups() | Drop any duplicate values during the creation of a unique index. |
+| background()| Create this index in the background.  There are some [considerations](http://docs.mongodb.org/manual/tutorial/build-indexes-in-the-background/#considerations) to keep in mind.
+| sparse() | Create a [sparse](http://docs.mongodb.org/manual/core/index-sparse/) index |
+| disableValidation() | By default, Morphia will validate field names being index.  This disables those checks. |
+| expireAfterSeconds() | Creates a [TTL Index](http://docs.mongodb.org/manual/core/index-ttl/) on a date field. |
+| fields() | This is the new way to define which fields to index. [Details]({{< ref "#Field" >}}) can be found below. |
+| options() | This is the new way to define index options. [Details]({{< ref "#IndexOptions" >}}) can be found below.  |
+
+#### Field
+```java
+ public @interface Field {
+     String value();
+     IndexType type() default IndexType.ASC;
+     int weight() default -1;
+ }
+```
+
+| Parameter           | Usage           |
+| ------------------- | --------------  |
+| value() | The field to include in the index |
+| type() | The type of index to create.  This includes the following values:   `ASC`, `DESC`, `GEO2D`, `GEO2DSPHERE`, `TEXT` |
+| weight() | When defining a text index, this is the weight to apply |
+
+#### IndexOptions
+```java
+public @interface IndexOptions {
+    String name() default "";
+    boolean unique() default false;
+    boolean dropDups() default false;
+    boolean background() default false;
+    boolean sparse() default false;
+    boolean disableValidation() default false;
+    int expireAfterSeconds() default -1;
+    String language() default "";
+    String languageOverride() default "";
+}
+```
+| Parameter           | Usage           |
+| ------------------- | --------------  |
+| name() | The name of the index |
+| unique() | Requires values in the index to be unique |
+| dropDups() | Drop any duplicate values during the creation of a unique index. |
+| background() | Create this index in the background.  There are some [considerations](http://docs.mongodb.org/manual/tutorial/build-indexes-in-the-background/#considerations) to keep in mind.
+| sparse() | Create a [sparse](http://docs.mongodb.org/manual/core/index-sparse/) index |
+| expireAfterSeconds() | Creates a [TTL Index](http://docs.mongodb.org/manual/core/index-ttl/) on a date field. |
+| disableValidation() | By default, Morphia will validate field names being index.  This disables those checks. |
+| language() | Default language for the index. |
+| languageOverride() | The field in the document to use to override the default language. |
+
+#### Indexed
+Applied to a Java field, marks the field to be indexed by mongodb.
+
+```java
+public @interface Indexed {
```

Comment:
This annotation wasn't updated to use the new stuff.  Probably wouldn't hurt to do so.  https://github.com/mongodb/morphia/issues/781